docs/rules/no-assignment-of-untracked-properties-used-in-tracking-contexts.md

```diff
@@ -69,6 +69,61 @@ class MyComponent extends Component {
 
 The autofixer for this rule will update assignments to use `set`. Alternatively, you can begin using tracked properties.
 
+## Configuration
+
+* object -- containing the following properties:
+  * array -- `extraMacros` -- array of configurations for custom macros, each with the following properties:
```

Comment:
Can you say `custom computed property macros which have dependent keys as arguments` to provide context about why macros matter to this rule?